src/main/python/api.py

```diff
@@ -0,0 +1,33 @@
+from flask import Flask, jsonify, request
+import argparse
+
+app = Flask(__name__)
+
+@app.route("/", methods=['GET'])
+def hello():
+    return "Hello! The server is working properly... :)"
+
+@app.route('/answer', methods=['POST'])
+def get_answer():
+    try:
+        req = request.get_json(force=True)
+        question = req["question"]
+        print("Question: %s" % (question))
+        # get the answer from the PyTorch model here
+        answer = "Albert Einstein (14 March 1879 – 18 April 1955) was a German-born theoretical physicist."
+        answer_dict = {"answer": answer}
+        return jsonify(answer_dict)
+    except Exception as e:
+        error_dict = {"error": "ERROR - could not parse the question or get answer."}
+        return jsonify(error_dict)
+
```

Comment:
The method works but as long as the API is followed. The Exception is there so that the API returns an error msg on some weird failure rather than exiting.